com.bilingify.readest.yml

```diff
@@ -0,0 +1,51 @@
+id: com.bilingify.readest
+
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+
+command: readest.sh
+rename-desktop-file: Readest.desktop
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=ipc
+  - --share=network
+
+modules:
+  - name: binary
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://github.com/readest/readest/releases/download/v0.9.63/Readest_0.9.63_amd64.deb
```

Comment:
why not building from source?

Reply:
The main challenge is that Readest relies on `pnpm`, which doesn't support offline installation of tarballs (https://github.com/flatpak/flatpak-builder-tools/issues/383#issuecomment-3459387435). This prevents a source build on Flathub. I hope we can get approval for a temporary repackaging, and we can switch to building from source once a solution for this `pnpm` limitation is available.

Reply:
OK